pymchelper/detector.py

```diff
@@ -129,7 +131,7 @@ def average_with_other(self, other_detector, error_estimate=ErrorEstimate.stderr
         self.data += delta / self.counter                      # mean += delta / n
         if error_estimate != ErrorEstimate.none:
             self._M2 += delta * (other_detector.data - self.data)  # M2 *= delta * (x - mean)
-            self.error = np.sqrt(self._M2 / (self.counter - 1))    # stddev = sqrt( var / (n-1) )
+            self.error = np.sqrt(self._M2 / (self.counter - 1))    # stddev = sqrt(1/(n-1)sum(x-<x>)**2)
```

Comment:
btw, you already provide the link in the header a bit further upstream the code.